### DISCUSSION TITLE
Using second order elemets but error message also occured

### DISCUSSION THREAD

**Discussion by killuaLeo** [neperfepx/FEPX#111] [General]
I used second order Hex element Using NEPER. However, when using FEPX to do simulation, the error message occurs that:
Error  :     > Mesh requires second-order elements.
Could anyone help me with this issue? Thanks!

**Comment by mkasemer** [comment]
@killuaLeo FEPX only uses 10-node (second-order) tetrahedral elements, per the documentation.

**Reply by killuaLeo** [reply]
_But when I tried to create tetrahedral mesh on NEPER, there is error information shown:_

neper -M 200-ori.tess -elttype tet -dim 3 -order 2 -cl 0.03

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.5.1-4
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 20 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/di/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] 200-ori.tess -elttype tet -dim 3 -order 2 -cl 0.03
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `200-ori.tess'...
Info   :     [i] Parsed file `200-ori.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 0.03) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0.76|0.9/65%|17%|18%)  
Info   :   - Fixing 2D-mesh pinches...
Info   :   - 3D meshing... Error  :   - Failed to read msh file
Aborted (core dumped)

_Could you please help me with this? Thanks a lot!_

**Reply by mkasemer** [reply]
This is a question for the Neper discussion forum, and I suggest including all commands (tessellation and meshing) to give the proper info.
